README.md

```diff
@@ -74,17 +76,26 @@ Next, update your `AppDelegate` to include the following function to allow the r
 import OktaAuth
 
 func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any]) -> Bool {
-  return OktaAuth.resume(url: url, options: options)
+  // oktaAuth - Configured OktaAppAuth instance used to start SignIn/SignOut flow. 
+  return oktaAuth.resume(url: url, options: options)
 }
 ```
 
 ## Usage Guide
 
-For an overview of this library's features and authentication flows, check out [our developer docs](https://developer.okta.com/code/ios).
+To start using our SDK you have to create a new object of  `OktaAppAuth`. You can instantiate  `OktaAppAuth` w/o parameters that means that SDK will use `Okta.plist` for configuration values. Alternatively you can create `OktaAppAuth` with custom configuration. For details about ways to configure `OktaAppAuth` see the [Configuration Reference](#configuration-reference)
+
+```swift
+import OktaAuth
 
-<!--
```

Comment:
Keep this along with the code block above please - we're adhering to a template.

Reply:
Done.